packages/react-composites/src/composites/ChatComposite/ChatScreen.tsx

```diff
@@ -170,7 +170,10 @@ export const ChatScreen = (props: ChatScreenProps): JSX.Element => {
     if (overlayImageItem === undefined) {
       return;
     }
-    const message = adapter.getState().thread.chatMessages[overlayImageItem?.messageId];
+    const message = adapter.getState().thread.chatMessages[overlayImageItem.messageId];
```

Comment:
nit: We can group this together with the above `if` condition since we need both to be true before we can do anything in here.